packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -331,9 +331,19 @@ async function generateBase64({ file, args = {}, reporter }) {
   if (options.duotone) {
     pipeline = await duotone(options.duotone, options.toFormat, pipeline)
   }
-  const { data: buffer, info } = await pipeline.toBuffer({
-    resolveWithObject: true,
-  })
+  let buffer
+  let info
+  try {
+    const result = await pipeline.toBuffer({
+      resolveWithObject: true,
+    })
+    buffer = result.data
+    info = result.info
+  } catch (err) {
+    reportError(`Failed to process image ${file.absolutePath}`, err, reporter)
```

Comment:
```suggestion
    reportError(`Failed to process image ${file.absolutePath}. 
It is probably corrupt, so please try replacing it.  If it still fails, please open an issue with the image attached.`, err, reporter)
```
Can we make the error more actionable? It may also be good to make the path relative to the site root, which would be easier to read.

Reply:
👍 to the changes. I'll note that all the other errors seem to use `absolutePath` — I can change that in a follow up PR.